Sources_v3/StreamChatUI/Utils/ChannelNamer.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+
+open class ChannelNamer {
```

Comment:
Why it needs to be class and not just function / closure?

Reply:
it's in-line with what we have in UIConfig (consistency)